extensions/whox.md

```diff
@@ -0,0 +1,96 @@
+---
+title: WHOX
+layout: spec
+copyrights:
+  -
+    name: "Simon Ser"
+    period: "2021"
+    email: "contact@emersion.fr"
+---
+
+## Introduction
+
+The `WHO` command allows clients to obtain information about other clients.
+However, `WHO` doesn't allow clients to request extra data (e.g. account names)
+nor a subset of the data (e.g. omit user and host).
+
+## `ISUPPORT` token
+
+Servers supporting this specification MUST include the `WHOX` token in
+`RPL_ISUPPORT` without any value. Clients MUST accept `WHOX` tokens with a
+value for forwards compatibility.
+
+## Extended `WHO` command
+
+The `WHO` command is extended with an optional argument:
+
+    WHO <mask> [%<fields>[,<token>]]
+
+The second argument contains a percent character followed by a list of fields
+requested by the client. Each field is represented by a letter. The standard
+fields are:
+
+- `t`: return the `<token>` specified by the client
```

Comment:
Incompatible with Insp: https://github.com/inspircd/inspircd/blob/ec37d6bb313f1c15df000fc635e0f2693209ddc7/src/coremods/core_who.cpp#L331-L332

Reply:
@progval you've linked to a (match) flag, the 't' field logic is compatible: https://github.com/inspircd/inspircd/blob/ec37d6bb313f1c15df000fc635e0f2693209ddc7/src/coremods/core_who.cpp#L423...L425